core/vgl/vgl_infinite_line_3d.hxx

```diff
@@ -16,8 +16,7 @@ vgl_infinite_line_3d<Type>::vgl_infinite_line_3d(vgl_point_3d<Type> const& p1,
 {
   vgl_vector_3d<Type> dir = p2-p1;
   vgl_infinite_line_3d<Type> l(p1, dir);
-  x0_ = l.x0();
-  t_ = dir;
+  *this = l;
```

Comment:
Would a delegating constructor be a more idiomatic way of expressing this?:
```
vgl_infinite_line_3d<Type>::vgl_infinite_line_3d(vgl_point_3d<Type> const& p1, vgl_point_3d<Type> const& p2)
: vgl_infinite_line_3d<Type>(p1, p2-p1)
{}
```

Reply:
I have updated the PR with your suggestion.